playkit/src/main/java/com/kaltura/playkit/PKTracksAvailable.java

```diff
@@ -0,0 +1,7 @@
+package com.kaltura.playkit;
+
+public enum PKTracksAvailable {
```

Comment:
@GouravSna  please rename PKTracksAvailableState/status

Reply:
Done changed it to PKTracksAvailableStatus